components/private/Icons/src/MDCIcons.m

```diff
@@ -32,7 +32,9 @@ + (nullable NSBundle *)bundleNamed:(nonnull NSString *)bundleName {
     NSBundle *baseBundle = [NSBundle bundleForClass:[MDCIcons class]];
     NSString *bundlePath = [baseBundle pathForResource:bundleName ofType:@"bundle"];
     bundle = [NSBundle bundleWithPath:bundlePath];
-    [bundleCache setObject:bundle forKey:bundleName];
+    if (bundle) {
```

Comment:
Please add a unit test to cover this case.

Reply:
Done.